toolkit/resources/manifests/package/pkggen_core_x86_64.txt

```diff
@@ -239,4 +239,4 @@ libselinux-3.2-1.cm2.x86_64.rpm
 slang-2.3.2-4.cm2.x86_64.rpm
 newt-0.52.21-3.cm2.x86_64.rpm
 chkconfig-1.20-2.cm2.x86_64.rpm
-msopenjdk-11-11.0.13+8-LTS-4.x86_64.rpm
+msopenjdk-11-11.0.13+8-LTS-4.x86_64.rpm
```

Comment:
nit - is this diff from a newline?

Reply:
I really don't know what this one is from, the git blame shows this entry getting added [here](https://github.com/microsoft/CBL-Mariner/commit/9d85614fabac7bb5092f17bbe5226e51c849d77f#diff-4399971797c1c0942b710773d3f583fb0c697de6dd1edc655a5b80e4e1f79462R240), so I'm pretty sure I didn't clip that change from his commit somehow. Probably safe to ignore?

Reply:
Just checked against main and the diff here looks to be caused by a newline. main has a newline at the end of the file while your change does not.

Reply:
```suggestion
msopenjdk-11-11.0.13+8-LTS-4.x86_64.rpm

```